site/src/server.rs

```diff
@@ -203,13 +203,18 @@ pub fn handle_dashboard(data: &InputData) -> dashboard::Response {
             }
 
             extend!(check_clean_points, r, r.is_clean() && r.check);
-            extend!(check_base_incr_points, r, r.is_base_incr() && r.check);
-            extend!(check_clean_incr_points, r, r.is_clean_incr() && r.check);
-            extend!(check_println_incr_points, r, r.is_println_incr() && r.check);
             extend!(debug_clean_points, r, r.is_clean() && !r.check && !r.release);
-            extend!(debug_base_incr_points, r, r.is_base_incr() && !r.check && !r.release);
-            extend!(debug_clean_incr_points, r, r.is_clean_incr() && !r.check && !r.release);
-            extend!(debug_println_incr_points, r, r.is_println_incr() && !r.check && !r.release);
+            // Skip these incremental measurements in versions prior to 1.24.0,
+            // because they didn't support incremental compilation.
+            if version != "1.19.0" && version != "1.20.0" && version != "1.21.0"  &&
+               version != "1.22.0" && version != "1.23.0" {
```

Comment:
I think we have code in the collector that does this -- we can parse the version as `semver::Version` and then compare as less than `1.24.0`. I think that should look cleaner.